keras_cv/quickstart_test.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+import pytest
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+import keras_cv
+
+"""
+ATTENTION!!!
```

Comment:
Perhaps slightly hacky but you could open the README or quickstart md files, concatenate together the `python` code blocks, and run that.

Reply:
That could work -- it does still leave the keras.io problem though.

Reply:
Ok I've kept thinking:

maybe keras.io could unit test the KerasCV quickstart using the same coding strategy?